Packs/rasterize/Integrations/rasterize/rasterize.py

```diff
@@ -248,22 +275,29 @@ def rasterize(path: str, width: int, height: int, r_type: RasterizeType = Raster
 
 
 def rasterize_webdriver(path: str, width: int, height: int, r_type: RasterizeType = RasterizeType.PNG, wait_time: int = 0,
-                        offline_mode: bool = False, max_page_load_time: int = 180, full_screen: bool = False):
+                        offline_mode: bool = False, max_page_load_time: int = 180, full_screen: bool = False,
+                        include_url: bool = False):
     """
-    Capturing a snapshot of a path (url/file), using Chrome Driver
+    Capturing a snapshot of a path (url/file), using Chrome Driver if include_url is set to False,
+    otherwise, it uses a virtual Display to display the screen of the linux machine.
+
     :param offline_mode: when set to True, will block any outgoing communication
     :param path: file path, or website url
     :param width: desired snapshot width in pixels
     :param height: desired snapshot height in pixels
     :param r_type: result type: .png/.pdf
     :param wait_time: time in seconds to wait before taking a screenshot
+    :param include_url: when set to True, will include the URL bar in the image result
     """
-    driver = init_driver(offline_mode)
+
+    driver, display = init_driver_and_display(width, height, offline_mode, include_url)
```

Comment:
There is a scenario that display will be setup and driver will fail. This line should be moved into the try/finally and quit_driver_and_display_and_reap_children should be modified to take into account that driver may be None or in partial state.

Reply:
done